### PR TITLE
3614_Create_Notification

### PR DESCRIPTION
**Pull request by dinhquang251**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -11,4 +11,4 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-API_KEY=KEY
+API_KEY=a7Ucuq0KY8Ksn8WzBG6wj4x6pcId6BpU
```
Comment (NguyenMinhTuanB) [inline gradle.properties]: không đưa key lên github

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by dinhquang251** [comment]
Create Notification play music
https://edu-redmine.framgia.vn/issues/3614

**Comment by dinhquang251** [comment]
ready

**Pull request merged**